playground/frontend/README.md

```diff
@@ -51,27 +48,49 @@ Run the following command to generate a release build:
 
 `$ flutter build web`
 
-### Tests
+This produces `build/web` directory with static files. Deploy them to your web server.
 
-Playground tests may be run using next commands:
+### Docker
 
-`$ flutter pub run build_runner build`
+To run the frontend app with Docker, run this in the frontend directory:
```

Comment:
Is Docker the only way to run this application?  Is there any way that we can include instructions for developers who want to contribute, but who are prevented for various reasons from using Docker?

May we consider adding instructions for these developers, including where it is expected to execute `flutter pub run build_runner build` such as in the playground_components folder?  I'm assuming such command runs in the context of the gradle command to setup the project, however, please see the errors I encountered when executing this setup command.

Reply:
Docker is not required. The recommended way to run this locally is still `flutter run`.
I updated the README stating the cases when one might prefer Docker.

Reply:
As for the local machine, I cannot reproduce this, and I cannot really see how is Flutter in $PATH for the terminal but not in $PATH for Gradle ran from that same terminal. Can you show the output of these?

1. `cat /etc/*-release`
2. `which flutter`
3. `echo $PATH`
4. `./gradlew :playground:frontend:printPath` (a new task I just added)